cannula/migrations/0026_ifasbottleneck.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
```

Comment:
Remove this line. It should only be necessary in Python2 and we are entirely Python3.

Reply:
commented out from __future__ import unicode_literals 0026_ifasbottleneck.py